schema/entity-record.json

```diff
@@ -54,7 +54,90 @@
           "description": "This may be used to provide a local name for this type of entity, or any further information to identify the type of entity. For example, in Finland 'ministeriö' for a government department."
         }
       },
-      "propertyOrder": 4
+      "propertyOrder": 4,
+      "allOf":[
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "arrangement"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "subtype": {
+                "enum":[
+                  "trust",
+                  "nomination",
+                  "other"
+                ]
+              }
+            }
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "legalEntity"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "subtype": {
+                "enum": [
+                  "trust",
+                  "other"
+                ]
+              }
+            }
+          }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "stateBody"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "subtype": {
+                "enum": [
+                  "governmentDepartment",
+                  "stateAgency",
+                  "other"
+                ]
+              }
+            }
+          }
+        },
+        {
+          "if": {
+            "not": {
```

Comment:
I think I'd just simplify this to: if 'registeredEntity' then 'other'.

We know that the entity codelist is likely to be simplified to remove 'registeredEntity' anyway in future, see #336 .

Reply:
I can switch this to a positive rather than a negative but will also need to include unknownEntity/anonymousEntity/state

Reply:
have done this

Reply:
Ahhh - yes. OK